app/models/ability.rb

```diff
@@ -143,6 +143,8 @@ def signed_in_with_organizer_role(user)
     can :manage, Vposition, conference_id: conf_ids_for_organizer
     can :manage, Vday, conference_id: conf_ids_for_organizer
     can :manage, Program, conference_id: conf_ids_for_organizer
+    can :manage, Schedule, program: { conference_id: conf_ids_for_organizer }
```

Comment:
I've already added tests for that
https://github.com/openSUSE/osem/pull/1094/files#diff-e9501dda6c23fd3e1076e2dff66d0f68R205